common/presenters/person-to-summary-list-component.test.js

```diff
@@ -46,42 +46,41 @@ describe('Presenters', function () {
         })
 
         it('should contain correct number of rows', function () {
-          expect(transformedResponse.rows.length).to.equal(6)
+          expect(transformedResponse.rows.length).to.equal(7)
         })
 
         it('should contain identifiers first', function () {
           const row1 = transformedResponse.rows[0]
           const row2 = transformedResponse.rows[1]
           const row3 = transformedResponse.rows[2]
-
+          row1 // ?
           expect(row1).to.deep.equal({
-            key: { html: '__translated__' },
+            key: { html: 'fields::police_national_computer.label' },
```

Comment:
These changes and all similar ones are because you've had to tweak things for the new `fields::category.uncategorised` tests. Gotcha :)

Reply:
I also thought it made it easier to understand what was going on rather than a blizzard of `__translated__`s ;)

Reply:
Yeah, definitely!